files-community/Files
# Feature: Added systray icon of Files shown when running on the background

## Summary

Users doesn't have a proper way to kill the app when running on the background. Added system tray icon to handle killing app by the app itself.

## Task Checklist

- [x] Added utility classes for the Taskbar
- [ ] Added functionalities to kill/resume

## Steps To Validate Changes

1. Launch the app.
2. Enable background process running.
3. Close window.
4. See notification area on the taskbar.

## Known Issues

_N/A_

## PR Checklist

- [x] **Development**: #14240

## Screenshots

![image](https://github.com/files-community/Files/assets/62196528/2bf50484-6895-4f09-9aae-76543a740e1a)

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
